WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -183,6 +183,15 @@ extension WooConstants {
         ///
         case productsFeedback = "https://automattic.survey.fm/woo-app-feature-feedback-products"
 
+        /// URL for the store setup feedback survey
+        ///
+        ///
```

Comment:
super nit: other cases have two comment lines?
```suggestion
```

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/9005/commits/ca9d6b9c8a0e1ebff431e4b18826ca8a69d09408